src/ingest-pipeline/airflow/dags/multiome.py

```diff
@@ -0,0 +1,336 @@
+from datetime import datetime, timedelta
+from pathlib import Path
+from typing import List
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.operators.dummy import DummyOperator
+from airflow.operators.python import BranchPythonOperator, PythonOperator
+from hubmap_operators.common_operators import (
+    CleanupTmpDirOperator,
+    CreateTmpDirOperator,
+    JoinOperator,
+    LogInfoOperator,
+    MoveDataOperator,
+    SetDatasetProcessingOperator,
+)
+
+import utils
+from utils import (
+    SequencingDagParameters,
+    get_absolute_workflows,
+    get_cwltool_base_cmd,
+    get_dataset_uuid,
+    get_parent_dataset_uuids_list,
+    get_parent_data_dirs_list,
+    build_dataset_name as inner_build_dataset_name,
+    get_previous_revision_uuid,
+    get_uuid_for_error,
+    join_quote_command_str,
+    make_send_status_msg_function,
+    get_tmp_dir_path,
+    pythonop_get_dataset_state,
+    HMDAG,
+    get_queue_resource,
+    get_threads_resource,
+    get_preserve_scratch_resource,
+)
+
+
+def generate_multiome_dag(params: SequencingDagParameters) -> DAG:
+    default_args = {
+        "owner": "hubmap",
+        "depends_on_past": False,
+        "start_date": datetime(2019, 1, 1),
+        "email": ["joel.welling@gmail.com"],
+        "email_on_failure": False,
+        "email_on_retry": False,
+        "retries": 1,
+        "retry_delay": timedelta(minutes=1),
+        "xcom_push": True,
+        "queue": get_queue_resource(params.dag_id),
+        "on_failure_callback": utils.create_dataset_state_error_callback(get_uuid_for_error),
+    }
+
+    with HMDAG(params.dag_id,
+               schedule_interval=None,
+               is_paused_upon_creation=False,
+               default_args=default_args,
+               user_defined_macros={
+                   "tmp_dir_path": get_tmp_dir_path,
+                   "preserve_scratch": get_preserve_scratch_resource(params.dag_id),
+               }) as dag:
+
+        cwl_workflows = get_absolute_workflows(
+            Path("salmon-rnaseq", "pipeline.cwl"),
```

Comment:
shouldn't the new multiome CWL needs to be here for processing?

Reply:
Good catch.  Just pushed a commit to address this